src/services/dataService/listItemFunctions.js

```diff
@@ -0,0 +1,41 @@
+const {ListItem} = require('./db')
+
+function findListItem(id) {
```

Comment:
This should be named `findListItems` and should be searching for a collection, not a single instance. Seems that findListItem(s) and getListItem are reversed.